supportedBackends/google/pipelines/v2alpha1/src/main/resources/transfer.sh

```diff
@@ -0,0 +1,168 @@
+# The `papi_v2_log` Centaur test is opinionated about the number of log messages around localization/delocalization.
+# The trace logging of `set -x` must be turned off for the `papi_v2_log` test to pass.
+set +x
+
+gsutil_log="gsutil_output.txt"
+
+
+localize_file() {
+  local cloud="$1"
+  local container="$2"
+  local rpflag="$3"
+  # Do not quote rpflag, when that is set it will be -u project which should be two distinct arguments.
+  rm -f "$HOME/.config/gcloud/gce" && gsutil ${rpflag} -m cp "$cloud" "$container" > "$gsutil_log" 2>&1
+}
+
+localize_directory() {
+  local cloud="$1"
+  local container="$2"
+  local rpflag="$3"
+  # Do not quote rpflag, when that is set it will be -u project which should be two distinct arguments.
+  mkdir -p "${container}" && rm -f "$HOME/.config/gcloud/gce" && gsutil ${rpflag} -m rsync -r "${cloud}" "${container}" > "$gsutil_log" 2>&1
+}
+
+# Content type is sometimes (not always) specified for delocalizations.
+gsutil_content_flag() {
+  local content="$1"
+
+  if [[ ! -z "${content}" ]]; then
+    echo -n -- "-h 'Content-Type: $content'"
+  fi
+}
+
+delocalize_file() {
+  local cloud="$1"
+  local container="$2"
+  local rpflag="$3"
+  local content="$4"
+
+  local content_flag=$(gsutil_content_flag $content)
```

Comment:
Looks like shellcheck has many comments along the lines of:
```
        ^-- SC2155: Declare and assign separately to avoid masking return values.
```

Reply:
I'll look at this in the next version, this happens to be a trouble spot.